packages/core/types/src/event-bus/common.ts

```diff
@@ -39,13 +41,12 @@ export type Message<T = unknown> = {
   options?: Record<string, unknown>
 }
 
-export type MessageFormat<T = unknown> = {
+export type RawMessageFormat<T = any> = {
```

Comment:
Why the rename, what is the non-raw message format?

Reply:
well it is the `Message`

Reply:
@adrien2p  Should we call it `RawMessage` without Format maybe?